src/eei.cpp

```diff
@@ -781,4 +781,18 @@ namespace HeraVM {
     }
     return false;
   }
+  size_t EthereumInterface::highestBitPosition(uint32_t value)
```

Comment:
Where is this needed?

Also, one could use `ffs(value)` (find first set) or `32 - clz(value)` (count leading zeroes) to get the same. These in many CPUs have their own instructions.

Reply:
oh nice,
nvm then